en/micro-integrator/docs/use-cases/examples/sequence_examples/using-multiple-sequences.md

```diff
@@ -144,6 +144,38 @@ Create the following mediation sequences:
 </sequence>
 ```
 
+As the backend create the following API which calls the stockquote backend service.
```

Comment:
"As the backend, create the following API which calls the `stockquote` backend service.